web/pdf_attachment_viewer.js

```diff
@@ -122,19 +121,13 @@ class PDFAttachmentViewer extends BaseTreeViewer {
     let attachmentsCount = 0;
     for (const name in attachments) {
       const item = attachments[name];
-      const content = item.content,
-        description = item.description,
-        filename = getFilenameFromUrl(
-          item.filename,
-          /* onlyStripPath = */ true
```

Comment:
Unless I'm mistaken I can't find other occurrences of `getFilenameFromUrl` that pass in the `onlyStripPath` argument, so should we perhaps remove it from `getFilenameFromUrl` as well?

Reply:
Oh, you're correct that it became unused; thank you!
I'll remove it in a separate commit, just to make it easier to restore the feature should the need ever arise.